examples/motorbike-application/templates/job-script-template.py.j2

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#SBATCH --partition {{ data.partition }}
+#SBATCH --nodes={{ data.nodes }}
+#SBATCH --ntasks={{ data.ntasks }}
+#SBATCH -J motorbike
+#SBATCH --output=/nfs/R-%x.%j.out
+#SBATCH --error=/nfs/R-%x.%j.err
```

Comment:
make sure to add in the docs this path must exist, **and**, if run in parallel, all nodes must have access to it

Reply:
This is a general problem with Slurm and Jobbergate. We don't know what paths the user will reference in their job scripts, so they can footgun themselves rather easily. We need to call it out in the docs. I will think about how to best approach that and include it in the PR for updating the Jobbergate docs.

In this case, the jobbergate-composed containers have a writable folder volume mounted at /nfs.